deployment/terraform/engine.go

```diff
@@ -40,6 +40,38 @@ func (t *Terraform) runCommand(dst io.Writer, args ...string) error {
 	mlog.Debug("Running terraform command", mlog.String("args", fmt.Sprintf("%v", args)))
 	cmd := exec.CommandContext(ctx, terraformBin, args...)
 
+	return _runCommand(cmd, dst)
+}
+
+func (t *Terraform) attachSG() error {
```

Comment:
Would it make sense to rename this to something a little more specific? As it is, it sounds like it could be used for any resource but it has a very limited use case right now.

Reply:
ok